deploy.sh

```diff
@@ -7,6 +7,7 @@ TF='./terraform'
 TMPDIR=${TMPDIR:-"/tmp"}
 LOGFILE=".ocp4-upi-powervs.log"
 source ./errors.sh
+DISTRO=$(lsb_release -ds 2>/dev/null || cat /etc/*release 2>/dev/null | head -n1 || uname -om || echo "")
```

Comment:
These commands are related to `Linux` if I am not wrong? Keep it at its original place.

Reply:
okay